website/static/js/nodeControl.js

```diff
@@ -265,6 +274,21 @@ function NodeControl (selector, data, options) {
 NodeControl.prototype.init = function() {
     var self = this;
     osfHelpers.applyBindings(self.viewModel, this.selector);
+    if (self.data.user.is_admin && !self.data.node.is_retracted) {
+        new NodesPrivacy('#nodesPrivacy', self.data.node, function(nodesChanged, requestedEmbargoTermination) {
+            // TODO: The goal here is to update the UI of the project dashboard to
+            // reflect the new privacy state(s). Unfortunately, since the components
+            // view is rendered server-side we have a relatively limited capacity to
+            // update the page. Once the components list is componentized we can
+            // rerender just that section of the DOM (as well as updating the
+            // project-specific UI).
+            // For now, this method only needs to handle requests for making embargoed
+            // nodes public.
+            if (requestedEmbargoTermination) {
```

Comment:
Yep, you're right; I realized my misunderstanding of this right after I posted my comment.